### PR TITLE
More functions in `prelude/set`.

### PR DESCRIPTION
**Pull request by jameshaydon**

### CODE REVIEW

File: rad/prelude/set.rad [hunk 1]
```diff
@@ -17,14 +21,31 @@
     (delete x s)))
 
 (def member?
-  "Query if an value is an element of a set."
-  (fn [x s]
-    (member? x s)))
+  "Query if a value is an element of a set."
```
Comment (jkarni) [inline rad/prelude/set.rad]: Looks like the fn def was lost?

File: rad/prelude/set.rad [hunk 2]
```diff
@@ -17,14 +21,31 @@
...
 
 (def to-vec
   "Convert a set to a vector."
-  (fn [s] (keys s)))
+  (fn [s] (dict/keys s)))
 
 (def from-seq
   "Create a set from a sequence."
-  (fn [xs] (foldr set/insert {} xs)))
+  (fn [xs] (foldr insert {} xs)))
```
Comment (jkarni) [inline rad/prelude/set.rad]: Maybe a test here (e.g., to check `(from-seq [1 2])` == `(from-seq [1 2 2])`))

File: rad/prelude/set.rad (hunk 1 above)
Comment (jameshaydon) [reply rad/prelude/set.rad]: Sets are implemented as dicts so `set/member?` is just an alias for `member?`.

Comment (jkarni) [reply rad/prelude/set.rad]: Hm, somehow didn't see the next line.

File: rad/prelude/set.rad [hunk 3]
```diff
@@ -17,14 +21,35 @@
...
+(def key-set
+  "The set of keys of a dict."
+  (fn [d]
+    (map-values (fn [v] #t) d)))
+
+(:test "key-set"
+  [ (key-set {:a 1 :b 2}) ==> {:a #t :b #t} ])
+
+(def subset?
+  "Checks if xs is a subset of ys."
```
Comment (MeBrei) [inline rad/prelude/set.rad]: Maybe add backticks around the variables for better visualization of the input.

**Comment by jameshaydon** [comment]
@jkarni @MeBrei bump

**Pull request merged**